test/specs/exampleUsingProject.e2e.ts

```diff
@@ -0,0 +1,15 @@
+import { step } from 'mocha-steps';
+import { TestSetup } from '../testSetup.ts';
+import * as utilities from '../utilities/index.ts';
+
+describe('an example using existing project', async () => {
+  let testSetup: TestSetup;
+
+  step('Set up the testing environment', async () => {
+    testSetup = new TestSetup('usingExistingProject');
+    await testSetup.setUp('developer', utilities.repoKeywords.DeployInv);
+    await utilities.reloadAndEnableExtensions();
+    await expect(1).toBe(false); // make it false to see screenshots
```

Comment:
I made the tests fail deliberately since I wanted to see if the project is really opened up.

Reply:
And the screenshots [here](https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/10466350108) tell.